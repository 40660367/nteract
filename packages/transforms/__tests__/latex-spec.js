import React from "react";
import renderer from "react-test-renderer";

import LaTeXDisplay from "../src/latex";

describe("LaTeXDisplay", () => {
  it("processes basic LaTeX", () => {
    const tree = renderer
      .create(<LaTeXDisplay data={"x^2 + y = 3"} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
