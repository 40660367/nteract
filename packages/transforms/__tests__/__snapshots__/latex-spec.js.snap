// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LaTeXDisplay processes basic LaTeX 1`] = `
<MathJaxNode
  inline={false}
  onRender={null}
>
  x^2 + y = 3
</MathJaxNode>
`;
